### PR TITLE
internal/testutil: add loggingexporter unmarshaller

### PR DESCRIPTION
**Pull request by gbbr**
Adds a new tool for reading metrics found in a collector log containing output from the `loggingexporter`. A CLI is also added which is able to take a log file along with a config and output the result of passing those metrics through the Datadog Exporter as JSON.

Usage:
```
Usage of /var/folders/jz/f8lwyqg9401_fd6cgpvfly0h0000gq/T/go-build446281604/b001/exe/testtool:
  -c string
    	Path to the collector config file; used for configuring the exporter. If left empty, default configuration will be used
  -l string
    	Collector log file output to use for input
  -t string
    	Ouput type: json or points (default "json")
  -v	verbose
```

Supports Metrics (Sum & Gauge). Can be extended to support Traces and other metric types.

### CODE REVIEW

File: exporter/datadogexporter/testtool.go [hunk 1]
```diff
@@ -0,0 +1,156 @@
...
+	"gopkg.in/yaml.v2"
+	"gopkg.in/zorkian/go-datadog-api.v2"
+
+	"github.com/open-telemetry/opentelemetry-collector-contrib/exporter/datadogexporter"
+	"github.com/open-telemetry/opentelemetry-collector-contrib/exporter/datadogexporter/internal/testutil"
+)
+
+const pathSeries = "/api/v1/series"
+
+func main() {
```
Comment (bogdandrutu) [inline exporter/datadogexporter/testtool.go:48]: We usually use cobra for commands.

**Review by bogdandrutu — COMMENTED** [review]
Why not in ./cmd?

File: exporter/datadogexporter/testtool.go (hunk 1 above)
Comment (mx-psi) [reply exporter/datadogexporter/testtool.go:48]: If there is interest on having this as a common utility, I think we should put it under `./cmd`, it should be useful to all, not only Datadog

Comment (gbbr) [reply exporter/datadogexporter/testtool.go:48]: If we want to make it available to everyone, I'd suggest making the [otlptext](https://github.com/open-telemetry/opentelemetry-collector/tree/main/exporter/loggingexporter/internal/otlptext) package exporter instead, and adding this as an `Unmarshal` method.

**Pull request closed without merging**